test/method.jl

```diff
@@ -112,6 +116,16 @@ end
         @test_signature ((::T) where T) -> 0   # Anonymous parameter
     end
 
+    @testset "callable structs" begin
+        ms = collect(methods(TestCallableStruct()))
+        sig1 = signature(filter(m -> m.nargs == 2, ms) |> first)
```

Comment:
Prefer to avoid `|>` where possible
```suggestion
        sig1 = signature(first(filter(m -> m.nargs == 2, ms))
```